packages/react/README.md

```diff
@@ -0,0 +1,31 @@
+# adCAPTCHA React
```

Comment:
To the uninitiated it's not really clear what this component is doing - some explanation would be good